skfp/preprocessing/filters/rule_of_veber.py

```diff
@@ -0,0 +1,82 @@
+from typing import Union
+
+from rdkit.Chem import Mol, rdMolDescriptors
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class RuleOfVeber(BaseFilter):
+    """
+    Rule of Veber
+
+    Rule designed to look for molecules that are likely to exhibit
+    good oral bioavailability. Described in [1]_.
```

Comment:
First line can probably be quite longer. Also, you can just cite [1]_ at the end of first sentence.